modules/tinymce/Gruntfile.js

```diff
@@ -807,8 +835,8 @@ module.exports = function (grunt) {
     'rollup',
     'unicode',
     'concat',
-    'uglify',
-    'copy'
+    'copy',
```

Comment:
I'm curious why you switched these?

Reply:
We need the copy task to run first, so it copies the icon pack to the dist directory. As the uglify step then runs on that copied file.